features/shared/checkout_previous_branch_after_success/same_current_branch/previous_branch_deleted.feature

```diff
@@ -0,0 +1,33 @@
+Feature: when the previous branch is deleted during a Git Town command, the main branch becomes the new previous branch
```

Comment:
Feature: deleting the current branch makes the main branch the new previous branch